src/dal/Jalasoft.TeamUp.Projects.DAL/ProjectsMongoDbRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Jalasoft.TeamUp.Projects.DAL
+{
+    using System;
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Projects.DAL.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using MongoDB.Driver;
+
+    public class ProjectsMongoDbRepository : IProjectsMongoDbRepository
+    {
+        private static MongoClient client;
+
+        public ProjectsMongoDbRepository()
+        {
+            client = new MongoClient("mongodb://localhost:27017");
```

Comment:
Maybe recive with dependency injection? I'm not sure

Reply:
`mongodb://localhost:27017`
You have this value in the config file you should be pulling it from there instead of hard coding it

Reply:
solved